pairing-tool/package.json

```diff
@@ -16,6 +16,15 @@
     "fliclib-linux-hci": "git+https://github.com/bravetechnologycoop/fliclib-linux-hci.git"
   },
   "devDependencies": {
-    "eslint": "^7.11.0"
+    "eslint": "^7.2.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-brave": "git+https://github.com/bravetechnologycoop/eslint-config-brave.git",
```

Comment:
Add version number to the end of this, 
`"eslint-config-brave": "git+https://github.com/bravetechnologycoop/eslint-config-brave.git#v1.0.0",`